### ISSUE TITLE
Kick and sliding doors

### ISSUE THREAD

**Issue by hawkwind3** [Realm667/WolfenDoom#1474]
Currently, the player can kick open sliding doors.
Realistically, this should not be the case.
Can this be addressed ?

**Comment by Tormentor667** [comment]
@AFADoomer implemented that feature, so he is the best one to ask for if this is even possible.

**Comment by Tormentor667** [comment]
Or maybe someone else is able to change the code in a way that sliding actions are not affected?

**Comment by hawkwind3** [comment]
Thanks to @Talon1024, all maps that use sliding poly doors cannot be kicked open. Closed.

**Comment by Talon1024** [comment]
I think there's a more significant underlying issue here. The kick can activate literally any line that can be used, including light switches and other things.

It's understandable for ventilation grates and the breakable mirror in C1M5, but otherwise, I don't think the kick should be able to activate any other line, other than the swinging doors.

**Comment by hawkwind3** [comment]
Agreed. But I have never used kick on switches, only on things that can be broken ... until now ...

**Comment by hawkwind3** [comment]
Reopened.

The latest weapons.zs causes the game to crash when kicking breakable things, like toilets and plants in pots.

![Image](https://github.com/user-attachments/assets/0375e55a-10d9-4e3a-93c5-d8efcce25bdb)

**Comment by x3003** [comment]
>I don't think the kick should be able to activate any other line, other than the swinging doors.

That's a shame, I use it all the time. I like it.
